src/Model/Repository/MyParcelConsignmentRepository.php

```diff
@@ -723,13 +727,23 @@ private function decodePickup($data)
             if (key_exists('location_name', $pickup)) {
                 $this->setPickupLocationName($pickup['location_name']);
             }
+
+            if (key_exists('location_code', $pickup)) {
+                $this->setPickupLocationCode($pickup['location_code']);
+            }
+
+            if (key_exists('retail_network_id', $pickup)) {
+                $this->setPickupNetworkId($pickup['retail_network_id']);
+            }
         } else {
             $this
                 ->setPickupPostalCode(null)
                 ->setPickupStreet(null)
                 ->setPickupCity(null)
                 ->setPickupNumber(null)
-                ->setPickupLocationName(null);
+                ->setPickupLocationName(null)
+                ->setPickupLocationCode(null)
+                ->getPickupNetworkId(null);
```

Comment:
The call to MyParcelConsignment::getPickupNetworkId() has too many arguments starting with null.